packages/backend/src/database/migrations/20220804115530_tile_hide_title_property.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from 'knex';
+
+const DASHBOARD_TILE_LOOMS_TABLE = 'dashboard_tile_looms';
+const DASHBOARD_TILE_CHARTS_TABLE = 'dashboard_tile_charts';
+
+const HideTitleColumn = 'hide_title';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.table(DASHBOARD_TILE_LOOMS_TABLE, (table) => {
+        table.string(HideTitleColumn);
```

Comment:
It's creating a string column instead of a boolean.
Do you prefer option value or making the column non nullable and with default false ? 
`table.boolean(...).notNullable().defaultTo(false);`

Reply:
You're right, good catch